{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -62,3 +62,11 @@ BABEL_DEFAULT_TIMEZONE = 'Europe/Zurich'
 THEME_FRONTPAGE_TITLE = "{{ cookiecutter.project_name }}"
 # Header logo
 THEME_LOGO="images/logo.svg"
+
+
+# Invenio-APP-RDM
+# ===============
+# See https://invenio-app-rdm.readthedocs.io/en/latest/configuration.html
+
+# Instance's theme entrypoint file. Path under `theme` folder.
+INSTANCE_THEME_FILE = 'theme.scss'
```

Comment:
Minor (see https://github.com/inveniosoftware/invenio-app-rdm/pull/59): what about `INSTANCE_THEME_SCSS` (or other variation)? (`FILE` is a bit generic ;) )